### PR TITLE
my first step to angular

### PR DESCRIPTION
**Pull request by timon94**

### CODE REVIEW

File: js/viewTranslations/viewTranslationsModule.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+(function(){
+    angular.module('viewTranlationsModule', []);
```
Comment (NE05KY) [inline js/viewTranslations/viewTranslationsModule.js]: Translations it's very confusing name for this feature.
Please rename it in Broadcast in the whole app

Comment (timon94) [inline js/viewTranslations/viewTranslationsModule.js]: done

File: js/viewBroadcasts/viewBroadcastsController.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+(function(){
+    angular.module('viewBroadcastsModule').controller('viewBroadcastsController', function(viewBroadcastsService, $scope) {
+        viewBroadcastsService.getData().then(function(value) {
+           $scope.transl = value;
```
Comment (NE05KY) [inline js/viewBroadcasts/viewBroadcastsController.js:4]: Forgotten variable

File: js/viewBroadcasts/viewBroadcastsService.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+(function() {
+    angular.module('viewBroadcastsModule').service('viewBroadcastsService', function($http) {
```
Comment (NE05KY) [inline js/viewBroadcasts/viewBroadcastsService.js:2]: Simplify name, just 'Broadcasts'

**Pull request merged**